arcgis-ios-sdk-samples/Display information/Custom dictionary style/CustomDictionaryStyleViewController.swift

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class CustomDictionaryStyleViewController: UIViewController {
+    /// The URL to the restaurants feature table.
+    let restaurantFeatureTableURL = URL(string: "https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/arcgis/rest/services/Redlands_Restaurants/FeatureServer/0")!
+    /// The URL to the symbol style dictionary from shared resources.
+    let restaurantStyleURL = Bundle.main.url(forResource: "Restaurant", withExtension: "stylx")!
+    
+    /// A feature layer that contains custom symbol style.
+    lazy var featureLayer: AGSFeatureLayer = {
```

Comment:
This may be better as a method that creates and returns the feature layer rather than a lazy property:

```swift
func makeFeatureLayer() -> AGSFeatureLayer { ... }
```

Once the feature layer is created, the map holds a reference to it, so the view controller doesn't need to. It also eliminates the potential retain cycle issues that can arise from the use of lazy properties. Lazy properties whose value relies on `self` should ideally capture an `unowned` reference to `self` so as to prevent a retain cycle in the event that the property is never read.